iosApp/Survey/Sources/Constants/Double+AnimationDuration.swift

```diff
@@ -12,6 +12,8 @@ extension Double {
 
     /// 0.7
     static var appearing: Self { 0.7.speedModified }
+    /// 0.01
+    static var instant: Self { 0.01.speedModified }
```

Comment:
I don't think we really need this value for an animation duration and it can be omitted. If we need to do an instant transition, then just triggering the UI event without any animation is much simpler and better, right? 🤔